site/content/docs/main/configuration.md

```diff
@@ -77,7 +77,8 @@ Where Contour settings can also be specified with command-line flags, the comman
 | default-http-versions     | string array           | <code style="white-space:nowrap">HTTP/1.1</code> <br> <code style="white-space:nowrap">HTTP/2</code> | This array specifies the HTTP versions that Contour should program Envoy to serve. HTTP versions are specified as strings of the form "HTTP/x", where "x" represents the version number.                                                                                              |
 | disableAllowChunkedLength | boolean                | `false`                                                                                              | If this field is true, Contour will disable the RFC-compliant Envoy behavior to strip the `Content-Length` header if `Transfer-Encoding: chunked` is also set. This is an emergency off-switch to revert back to Envoy's default behavior in case of failures.
 | disableMergeSlashes       | boolean                | `false`                                                                                              | This field disables Envoy's non-standard
-merge_slashes path transformation behavior that strips duplicate slashes from request URL paths.
+| serverHeaderTransformation       | string                | `OVERWRITE`                                                                                              | This field defines the action to be applied to the Server header on the response path. Values: `OVERWRITE` (default), `APPEND_IF_ABSENT`, `PASS_THROUGH`
+merge_slashes path transformation behavior that strips duplicate slashes from request URL paths. 
```

Comment:
Looks like this line needs to move up above the `serverHeaderTransformation` entry